cypress/integration/prospects/profile.js

```diff
@@ -24,4 +24,62 @@ describe('Prospects profile info', () => {
     cy.get('.error-title').contains('User not found')
     cy.get('.error-message').contains('The user you are searching for does not exist.')
   })
-})
\ No newline at end of file
+
+  it(`User should be able to undo prospect deletion by canceling during
```

Comment:
Now we talking, about great tests! However, I see that you reversed the order of the tests, so, if you only test for "cancel" the action, then you can go from 1 to 10 in both tests because you will delete the prospects after you check the cancel button.